pkg/asset/manifests/vsphere/cloudproviderconfig.go

```diff
@@ -32,9 +32,10 @@ func CloudProviderConfigYaml(infraID string, p *vspheretypes.Platform) (string,
 			vCenterPort = vCenter.Port
 		}
 		vCenterConfig := cloudconfig.VirtualCenterConfigYAML{
-			VCenterIP:   vCenter.Server,
-			VCenterPort: uint(vCenterPort),
-			Datacenters: vCenter.Datacenters,
+			VCenterIP:    vCenter.Server,
+			VCenterPort:  uint(vCenterPort),
+			Datacenters:  vCenter.Datacenters,
+			InsecureFlag: true,
```

Comment:
Could you explain the reason why the InsecureFlag is now to true?

Reply:
I changed this to true to match the INI config below.  
`printIfNotEmpty(buf, "insecure-flag", "1")`
For some reason we had YAML to false, but INI to true.

Reply:
The insecure flag should be set to `true`. 
While installing on vSphere requires a certificate running on vSphere doesn't and just uses insecure. If we were going to move to insecure `false` this it would be a significant change.